### PR TITLE
OCPBUGS-43830: Expand "Rollback Helm Release" status width

### PR DESCRIPTION
**Pull request by logonoff**
**Fixes**: 

https://issues.redhat.com/browse/OCPBUGS-43830

**Analysis / Root cause**: 

Status text was too long for the width so it overlapped with the content in the next column

**Solution Description**: 

Expand width of status header

**Screen shots / Gifs for design review**: 

Before:
Screenshot From 2024-11-20 10-16-38

After:
Screenshot From 2024-11-20 10-16-34

**Unit test coverage report**: 

n/a

**Test setup:**

Create a helm release, then upgrade it, then go to "Rollback Helm Release"

**Browser conformance**: 

- [x] Chrome
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/Status.tsx [hunk 1]
```diff
@@ -8,13 +8,12 @@ import {
...
 } from '@patternfly/react-icons';
-import { StatusComponentProps } from '../../../extensions/console-types';
 import { DASH } from '../../constants';
 import { YellowExclamationTriangleIcon } from './icons';
 import { ErrorStatus, InfoStatus, ProgressStatus, SuccessStatus } from './statuses';
-import StatusIconAndText from './StatusIconAndText';
+import { StatusIconAndText, StatusIconAndTextProps } from './StatusIconAndText';
 
-export type StatusProps = StatusComponentProps & {
+export type StatusProps = StatusIconAndTextProps & {
```
Comment (logonoff) [inline frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/Status.tsx:16]: all paths of this component end up returning a `StatusIconAndText` so this is fine to do

**Review by lokanandaprabhu — CHANGES_REQUESTED** [review]
Hi @logonoff ,

Instead of this change, we can achieve this by just changing status classname from

`  status: 'pf-m-hidden pf-m-visible-on-lg pf-v5-u-w-8-on-xl pf-v5-u-w-16-on-lg',
`  
  to
  
`    status: 'pf-m-hidden pf-m-visible-on-lg pf-v5-u-w-16-on-xl pf-v5-u-w-16-on-lg',
`

I am adding the result of the above change, 

https://github.com/user-attachments/assets/a2d85629-b6b0-453f-b727-a13a4b8fabf2

**Review by lokanandaprabhu — APPROVED** [review]
/lgtm

**Pull request merged**